### DISCUSSION TITLE
Does Helix supports all predicates and directives that's defined in tree-sitter docs?

### DISCUSSION THREAD

**Discussion by asumbek** [helix-editor/helix#12343] [Q&A]
Pretty much self-explanatory, does Helix supports all that's defined in [Predicates and Directives](https://tree-sitter.github.io/tree-sitter/using-parsers/queries/3-predicates-and-directives.html) section?

**Comment by the-mikedavis** [comment]
I know off the top of my head that we support at least `#eq?`, `#match?`, `#any-of?`, `#is?`, the `not-` versions of those and `#set!`, I'm not sure about the others listed there. As mentioned in the note at the bottom of that page, predicates are handled in the bindings rather than the tree-sitter C library, so any ones that are missing would need to be added to `helix-core/src/syntax.rs`.
